doc/classes/Rect2.xml

```diff
@@ -133,8 +133,10 @@
 			</return>
 			<argument index="0" name="b" type="Rect2">
 			</argument>
+			<argument index="1" name="borders_touch" type="bool" default="false">
+			</argument>
 			<description>
-				Returns [code]true[/code] if the [Rect2] overlaps with another.
+				Returns [code]true[/code] if the [Rect2] overlaps with another or (If [code]borders_touch[/code] is set to [code]true[/code]) if their borders enter into contact.
```

Comment:
```suggestion
				Returns [code]true[/code] if the [Rect2] overlaps with [code]b[/code] (i.e. they have at least one point in common).
				If [code]borders_touch[/code] is [code]true[/code], they will also be considered overlapping if their borders touch, even without intersection.
```

Reply:
BTW it could also be `include_borders` instead of `borders_touch`. I don't have a strong opinion either way, just mentioning it.

Reply:
Changes made.